episodes/chapter6.md

```diff
@@ -0,0 +1,570 @@
+2024-05-24 15:52:13 - context        - INFO     -                   _                     __
+2024-05-24 15:52:13 - context        - INFO     -                  | |                   / /
+2024-05-24 15:52:13 - context        - INFO     - __   ____ _ _ __ | |_ __ _  __ _  ___ / /_
+2024-05-24 15:52:13 - context        - INFO     - \ \ / / _` | '_ \| __/ _` |/ _` |/ _ \ '_ \
+2024-05-24 15:52:13 - context        - INFO     -  \ V / (_| | | | | || (_| | (_| |  __/ (_) |
+2024-05-24 15:52:13 - context        - INFO     -   \_/ \__,_|_| |_|\__\__,_|\__, |\___|\___/
+2024-05-24 15:52:13 - context        - INFO     -                             __/ |
+2024-05-24 15:52:13 - context        - INFO     -                            |___/
+2024-05-24 15:52:13 - context        - INFO     -
+2024-05-24 15:52:13 - context        - INFO     -  --> Join us on Discord! https://discord.gg/rwRvwyK
+2024-05-24 15:52:13 - context        - INFO     -  --> Docs: https://docs.vantage6.ai
+2024-05-24 15:52:13 - context        - INFO     -  --> Blog: https://vantage6.ai
+2024-05-24 15:52:13 - context        - INFO     - ------------------------------------------------------------
+2024-05-24 15:52:13 - context        - INFO     - Cite us!
+2024-05-24 15:52:13 - context        - INFO     - If you publish your findings obtained using vantage6,
+2024-05-24 15:52:13 - context        - INFO     - please cite the proper sources as mentioned in:
+2024-05-24 15:52:13 - context        - INFO     - https://vantage6.ai/vantage6/references
+2024-05-24 15:52:13 - context        - INFO     - ------------------------------------------------------------
+2024-05-24 15:52:13 - context        - INFO     - Started application vantage6
+2024-05-24 15:52:13 - context        - INFO     - Current working directory is '/Users/clgeng/github/advantage/vantage6-workshop'
+2024-05-24 15:52:13 - context        - INFO     - Successfully loaded configuration from '/Users/clgeng/Library/Application Support/vantage6/node/node1.yaml'
+2024-05-24 15:52:13 - context        - INFO     - Logging to '/Users/clgeng/Library/Logs/vantage6/node/node1/node_user.log'
+2024-05-24 15:52:13 - context        - INFO     - Common package version '4.1.0'
+2024-05-24 15:52:13 - context        - INFO     - vantage6 version '4.1.0'
+[info ] - Configuration file = ***/vantage6/node/node1.yaml
+[info ] - Log file           = ***/vantage6/node/node1/node_user.log
+[info ] - data folders       = ***/vantage6/node/node1
+[info ] - Database labels and files
+[info ] -  - default         = ***/data.csv (type: csv)
+[info ] -  - default         = ***/data.sql (type: sql)
+```
+
+You can see that not only the path of the configuration file is printed out, but also the locations of the log file, the data folders and the database files are shown.
+
+::::::::::::::::::::::::::::::::::::: challenge
+
+## Challenge 2: Create a new node configuration
+
+1. Create a new node configuration using the `v6 node new` command.
```

Comment:
Since we are also making them start the node in another exercise below, we should provide them with an API key here. Or they could create one themselves from the UI (which they have learned the day before in lesson 4 I think)?

Maybe create another issue for this as it's not really lesson development but more infra preparation?

Reply:
Yes, we briefly discussed this with @CunliangGeng and we thought that we can setup a server for the workshop, create an organization per participant and give them an API key. Also because maybe someone is attending the second day, but not the first. I will open an issue for this.